src/migrations/utils.ts

```diff
@@ -14,8 +13,7 @@ export function getDuplicatedIds<Obj extends Ref>(objects: Obj[]): string[] {
 }
 
 export function getMigrationsForNode(): Migration[] {
-    const directory = path.join(__dirname, "tasks");
-    const keys = _.sortBy(fs.readdirSync(directory));
+    const keys = _.sortBy(fs.readdirSync("./src/migrations/tasks"));
```

Comment:
@tokland Relative path on static code analysis doesn't work and it broke the functionality to merge all the code into a single distributable file.

Reply:
What do you propose? Originally, I had explicit imports, but I wanted to avoid the hassle both add the file and add the import + some array.

Reply:
> What do you propose? Originally, I had explicit imports, but I wanted to avoid the hassle both add the file and add the import + some array.

Actually the snipped I changed (relative from project root) seems to work just fine, however I wanted your feedback on this change.

Reply:
Oh, ok, I misunderstood :+1: